Server/controllres/giphy.js

```diff
@@ -0,0 +1,6 @@
+const path=require('path')
+const getGiphy=(req, res) => {
```

Comment:
`getGiphyPage` would make more sense, reading `getGiphy` I expect it to return a list of giphy or something like that